debian/control

```diff
@@ -74,7 +74,7 @@ Recommends: default-dbus-system-bus | dbus-system-bus,
 Suggests: systemd-container,
           systemd-homed,
           systemd-userdbd,
-          systemd-boot,
+          systemd-boot-efi-signed | systemd-boot-efi,
```

Comment:
I don't think this part is necessary. I think we still would want `systemd` to suggest the full `systemd-boot` package. You're handling the `efi` vs `efi-signed` below.

Reply:
I went ahead and removed that.